### ISSUE TITLE
Bring back src query parameter to default WDC loaded in simulator

### ISSUE THREAD

**Issue by iamEAP** [tableau/webdataconnector#105]
What happened to the `src` query parameter in the simulator auto-populating the URL?  Looks like it was documented in #82, but then got removed somewhere along the way.

It'd be super handy to bring back, I'm trying to standardize a few things in the Web Data Connector Generator like so:

``` sh
# Runs npm start on WDC, npm start on Simulator, opens simulator + WDC in default browser
npm run simulate

# Same, but instead of npm start on the WDC, it runs grunt / grunt watch
npm run simulate:dev
```

**Comment by jdomingu** [comment]
As part of the simulator improvements, the team added an MRU list. I might be wrong, but I believe this was intentionally removed.

**Comment by lbrendanl** [comment]
Thanks @iamEAP, we had removed it when Austin added the MRU and cookie feature.   We did not consider this use case though.

@jagreene can you add the src param back in so we can support this?

**Comment by iamEAP** [comment]
Would be much appreciated! 🙏

**Comment by jagreene** [comment]
Should be resolved with #108 when it is merged
